.github/workflows/examples-mbed.yaml

```diff
@@ -47,6 +47,19 @@ jobs:
               with:
                   submodules: true
 
+            - name: Bootstrap
+              timeout-minutes: 10
+              run: scripts/build/gn_bootstrap.sh
+
+            - name: Uploading bootstrap logs
+              uses: actions/upload-artifact@v2
+              if: ${{ always() }}
+              with:
+                  name: bootstrap-logs
+                  path: |
+                   .environment/gn_out/.ninja_log
+                   .environment/pigweed-venv/*.log
+
             - name: Build lock-app example
               timeout-minutes: 10
```

Comment:
if the timeout is 10 minutes, should we not increase this side?

I imagine now it would work better because bootstrap will be split out of the lock example build,

Reply:
I think the more consistent with the whole project is a separate bootstrap step. Now lock example building should take about 5 minutes.